src/torznab.ts

```diff
@@ -256,6 +258,14 @@ async function createTorznabSearchQueries(
 	}
 }
 
+export async function getSearchString(searchee: Searchee): Promise<string> {
```

Comment:
It could use ids but the issue is that the parse endpoint for the arrs only takes one item at a time. So we would likely have to cache it. For fresh users we would need to spend a couple minutes slowly building it for first time users before any searching can take place.

I don't expect much issues with the current. The groups are guaranteed to be valid, that is no item is hitting the wrong cache. The only issue that ids would solve is maybe grouping items that are badly named or for anime. So I don't think it's worth using ids.